main/src/library/Vector.flix

```diff
@@ -26,7 +26,7 @@ instance Indexable[Vector[a]] {
     type Elm = a
     type Aef = OutOfBounds
 
-    pub def get(i: Int32, t: Vector[a]): a \ OutOfBounds = {
+    pub def get(t: Vector[a], i: Int32): a \ OutOfBounds = {
```

Comment:
I would not swap the arguments for Array and Vector etc. That causes downstream breakages. I was only thinking of swapping them for the traits.

Reply:
Yeah this is only on the instances

Reply:
Ah. I saw wrong.